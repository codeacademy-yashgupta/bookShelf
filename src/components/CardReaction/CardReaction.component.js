import React, { Component } from 'react';
import './CardReaction.css';

class CardReaction extends Component {
  render() {
    return (
      <div className="card-reaction">
        <img src="./icons/heart-black.svg" className="icon-heart" alt="Press for like" />
      </div>
    );
  }
}


export default CardReaction;
