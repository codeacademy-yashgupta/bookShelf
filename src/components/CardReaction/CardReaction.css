.icon-heart {
  height: 25px;
  width: 25px;
  padding: 10px;
}