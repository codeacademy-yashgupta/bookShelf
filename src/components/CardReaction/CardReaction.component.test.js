import React from 'react';
import renderer from 'react-test-renderer';
import CardReaction from './CardReaction.component';

describe('CardReaction', () => {
  it('should render like button', () => {
    const tree = renderer.create(<CardReaction />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
