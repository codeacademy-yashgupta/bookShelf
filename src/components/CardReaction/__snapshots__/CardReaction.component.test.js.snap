// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardReaction should render like button 1`] = `
<div
  className="card-reaction"
>
  <img
    alt="Press for like"
    className="icon-heart"
    src="./icons/heart-black.svg"
  />
</div>
`;
